Sources/ChatImpl.swift

```diff
@@ -608,22 +604,6 @@ extension ChatImpl: Chat {
     }
   }
 
-  public func getThreadChannel(
-    message: MessageImpl,
```

Comment:
Shouldn't be getThreadChannel somewhere in a project?

Reply:
I see `getThreadChannel` is in the `ChatInternal.kt` file. Do your customers can ever invoke `getThreadChannel`? It's not part of the `pubnub-chat` project. 

Wojtek told me that the visible symbols for the customer are placed in `pubnub-chat`, so they don’t see anything from `pubnub-chat-impl`, where ChatInternal is located

Reply:
In KMP `getThreadChannel` is hidden from customers. Its implementation serves as implementation for Message#getThread

`override fun getThread() = chat.getThreadChannel(this)`

That's way I asked about presence of getThreadChannel in swift.  But it seem that Message#getThread is implemented differently in Swift so we are good.